src/Redux/Books/Books.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+
```

Comment:
- Please Define action types for `adding` and `removing` a book:
- it can be something similar like this: #const ADD_BOOK = bookstore/books/ADD_BOOK and #const REMOVE_BOOK = bookstore/books/REMOVE_BOOK